constructorio-client/src/main/java/io/constructor/client/ConstructorIO.java

```diff
@@ -1006,7 +1002,7 @@ protected HttpUrl makeUrl(List<String> paths) throws UnsupportedEncodingExceptio
             .addQueryParameter("key", this.apiKey)
             .host(this.host);
 
-        if (!paths.contains("catalog")) {
+        if (!paths.contains("catalog") && !paths.contains("tasks") && !paths.contains("task")) {
```

Comment:
So Paths is a List/Array currently so I would have to loop through and match each path. I came up with a way below but it might make it a little less readable.

` paths.stream().anyMatch( path -> Pattern.matches("(catalog|task?)", path));`

Not too familiar with java, I haven't tested this but do you think I should try it? Could be a better way I'm not seeing.

Reply:
Ooo, didn't realize it was a list. Let's leave it as is for now :)